test/util/collect.js

```diff
@@ -7,7 +7,7 @@ import { diff } from "jsondiffpatch";
  * Collects test items from the specified directory
  * @param {string} base
  */
-export function* collect(base, { expectError, raw } = {}) {
+export function* collect(base, { expectError, raw, extensions = {} } = {}) {
```

Comment:
(I guess this is not needed anymore)

Reply:
Addressed in [1b29589](https://github.com/w3c/webidl2.js/pull/731/commits/1b2958989a9fd81796d82b668b112222ef0d1648).